extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/QuarkusErrorHandler.java

```diff
@@ -110,15 +110,21 @@ public void accept(Throwable throwable) {
                     .append("\",\"stack\":\"")
                     .append(escapedStack)
                     .append("\"}");
-            event.response().end(jsonPayload.toString());
+            writeResponse(event, jsonPayload.toString());
         } else {
             //We default to HTML representation
             event.response().headers().set(HttpHeaderNames.CONTENT_TYPE, "text/html; charset=utf-8");
             final TemplateHtmlBuilder htmlBuilder = new TemplateHtmlBuilder("Internal Server Error", details, details);
             if (showStack && exception != null) {
                 htmlBuilder.stack(exception);
             }
-            event.response().end(htmlBuilder.toString());
+            writeResponse(event, htmlBuilder.toString());
+        }
+    }
+
+    private void writeResponse(RoutingContext event, String output) {
```

Comment:
Should method be `static` as the others?

Reply:
It doesn't make a difference, it's fine being an instance method for now